install.sh

```diff
@@ -27,7 +27,10 @@ sudo ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws
 rm awscli-bundle.zip
 
 git clone https://github.com/articulate/docker-consul-template-bootstrap.git
+cd docker-consul-template-bootstrap
```

Comment:
`cd`ing in and then back out seems error prone to me, might consider setting this dir to a variable and using that var in your commands?